pages/text-encoding.xml

```diff
@@ -480,6 +480,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             other</p>
                      </cell>
                   </row>
+                  <row>
+                     <cell>ancient correction</cell>
+                     <cell>
+                     <egXML xmlns="http://www.tei-c.org/ns/Examples">
+                        <choice>
+                        <orig>ቦ</orig>
+                        <corr>በ</corr>
+                     </choice>
+                     </egXML>
+                     </cell>
+                     <cell>
+                        <p>this will print the corrected version in {}</p>
```

Comment:
Is the original reading is also displayed?

Reply:
not in the app at the moment. it may be in the apparatus and in print.

Reply:
okay, wonderfl! Thank you!
Sorry to ask, but this will be in the guidelines as well?
how do I encode this now?
Orig: ልቦ፡
Corrected ልበ፡